workers/docker-worker/schemas/v1/payload.yml

```diff
@@ -123,6 +123,17 @@ properties:
           still includes feature in order for d2g to work with it.
         type: boolean
         default: false
+      containerEngine:
+        title: Container engine
+        description: >-
+          The container engine to use, between `docker` and `podman`.
+          This property is only used during Generic Worker's internal
+          payload translation (d2g). Default is `podman`.
+        type: string
+        enum:
+          - docker
+          - podman
+        default: podman
```

Comment:
Tough choice probably as there are already so many pools on community-tc converted to d2g, but having major fxci upgrade ahead maybe it's worth making `docker` default? 
Although it might require changes on the taskgraph side as well, so doesn't really matter?

Reply:
Maybe we can tie @taskcluster/releng in here and get their opinion.

Reply:
Sounds like having a worker config setting is preferred. I'll keep this payload change as well for quick toggling in individual tasks.

Reply:
It sounds like maybe you've moved past this already, but fwiw, I don't think either default would be a big deal. Any migrations to g-w/d2g that we do will involve a bunch of testing on Try or in PRs, and we can fairly easily set this to whatever when needed.